README.md

```diff
@@ -18,6 +29,58 @@ Or install it yourself as:
 
 ## Usage
 
+### Configuration in Rails
+
+The goal: constrain all configuration regarding the logging to go through
+logatron and not the internals of logatron (such as lograge). Consumers of
+logatron should not have to know the internals.
+
+Logatron must be configured before anything else since there can be other
+configuration dependent upon those values (lograge for sure). This is done by
+doing that configuration in an initializer file that will be listed first when
+sorted in that directory alphanumerically. Putting an underscore (`_`) in front
+of the name typically will move it to the top of the list - 
+`config/initializers/_log.rb`. Here is an example configuration.
+
+```ruby
+require 'logatron/logatron'
+
+Logatron.configure do |config|
+  log_name = 'quartermaster'
+  if ENV['DAEMON']
+    script  = File.basename(ENV['DAEMON'])
+    script.slice! File.extname(script)
+    log_name = script
+  end
+  destination = Rails.env == 'test' ? '/dev/null' : STDOUT
+  config.host = `hostname`.chomp
+  config.app_id = log_name
+  config.logger = Logger.new(destination)
+  config.level = Logatron::INFO
+  config.base_controller_class = 'ActionController::API'
+  config.add_rails_request_field(:user_agent, &:user_agent)  # optional
+end
+```
+
+The `ENV['DAEMON']` logic is for Ascent when deploying as an AMQP daemon
+instead of a rails app (outside the scope of this README.md).
```

Comment:
The example config doesn't need to show that logic. Just show that `config.app_id` should be set to a string identifying the current app. Remove the `if` and inline `log_name`.